source/ios/AdaptiveCards/AdaptiveCards/AdaptiveCards/ACRInputLabelView.h

```diff
@@ -29,6 +29,7 @@
 @property IBInspectable UIColor *validationFailBorderColor;
 @property IBInspectable CGFloat validationFailBorderRadius;
 @property IBInspectable CGFloat validationFailBorderWidth;
+@property (nonatomic, strong) NSArray *targetInputIds;
```

Comment:
this should be private property? Should be available only in .mm file.

Reply:
removed instance variable.